DOCKER.md

```diff
@@ -0,0 +1,97 @@
+# Using Docker and Docker Compose to run OpenStreetMap
+
+Using [Docker](https://www.docker.com/) will allow you to install the OpenStreetMap application and all its dependencies in Docker images and then run them in containers, almost with a single command. You will need to install Docker and Docker Compose on your development machine:
+
+- [Install Docker](https://docs.docker.com/install/)
+- [Install Docker Compose](https://docs.docker.com/compose/install/)
+
+These instructions gloss over the precise details of the dependencies and their configuration but these can be found in full detail at [INSTALL.md](INSTALL.md).
+
+The first step is to fork/clone the repo to your local machine. The repository is reasonably large (~150MB) and it's unlikely that you need the full history. If you are happy to wait for it all to download, run:
+
+    git clone https://github.com/openstreetmap/openstreetmap-website.git
+
+To clone only the most recent version (~23MB), instead use a 'shallow clone':
+
+    git clone --depth=1 https://github.com/openstreetmap/openstreetmap-website.git
+
+Now change working directory to the `openstreetmap-website`:
+
+    cd openstreetmap-website
+
+### Storage setup
+
+    cp config/example.storage.yml config/storage.yml
+
+### Database
+
+    cp config/docker.database.yml config/database.yml
+
+### App configuration
+
+    cp config/settings.yml config/settings.local.yml
```

Comment:
No, please don't do this. The settings.local.yml file should only contain values that you want to override.

See https://github.com/openstreetmap/openstreetmap-website/blob/edd49e85823079d6de19cbc9f89c6abc0568382b/INSTALL.md#prepare-local-settings-file

Reply:
👍 Fixed by copying content from `INSTALL.md`. 

For the record, I don't really like all the duplication of content from `INSTALL.md` but having a complete set of Docker-specific instructions seems valuable instead of making people piece together instructions from multiple files. If someone has a better idea on how to reduce the duplication please let me know.